com.microsoft.mrtk.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -418,14 +434,14 @@ private void Update()
                     }
                 }
 
-                foreach (XRBaseController controller in destroyedControllers)
+                foreach (GameObject controller in destroyedControllers)
                 {
                     controllerMapping.Remove(controller);
                 }
 
                 foreach (IXRInteractor interactor in destroyedInteractors)
                 {
-                    if (interactor is XRBaseControllerInteractor controllerInteractor)
+                    if (interactor is XRBaseInteractor controllerInteractor)
```

Comment:
This isn't a `controllerInteractor` anymore (also this cast might not need to exist?)

Reply:
Fixed